design/foundation-proposal.md

```diff
@@ -0,0 +1,135 @@
+
+https://www.cncf.io/sandbox-projects/
+
+## Motivation
+
+When we started the metal3-io project in early 2019, we skipped over doing much
+to formalize governance.  We felt focusing on building something useful was
+more important.  If the project worked out and we started building a community,
+then we would revisit formalizing project governance for the future.
+
+After a year, we have produced useful code and a community involving multiple
+companies has formed to collaborate.  It’s a good time to consider what changes
+might best support this community for the future.
+
+Some changes have already started happening, such as formalizing the process
+for adding and removing committers to repositories. This document suggests
+a next step of applying as a CNCF Sandbox project.
+
+### Goals
+
+* Align with other CNCF ecosystem projects in terms of policies and processes.
+* Assets are transferred to a neutral third party (i.e. domains)
+* Gain more visibility in the CNCF ecosystem to attract others who may be
+  interested in collaborating on bare metal host provisioning with Kubernetes.
+
+### Non-Goals
+
+* No changes to the metal3-io code are in scope of this proposal.  This
+  proposal is only about the community future.
+* No changes to committer processes are proposed here.  We already have a
+  documented [process for adding and removing
+  committers](../maintainers/README.md).
+
+## Proposal
+
+Apply to become a CNCF Sandbox project.
+
+https://www.cncf.io/sandbox-projects/
+
+The [Goals](#goals) section discusses some of the immediate changes we should
+expect, but as the project continues to mature, many of the other services
+provided by the CNCF would be very beneficial to metal3-io.  The CNCF web site
+does a nice job discussing what services they offer projects on the
+[Services for CNCF Projects](https://www.cncf.io/services-for-projects/) page.
+
+## Design Details
+
+### Work Items
+
+* (russellb) Write this proposal.
+* (everyone) Provide feedback on initial proposal, reach consensus.
+* (russellb) If approved, draft the [CNCF Sandbox
+  application](https://github.com/cncf/toc/blob/master/process/project_proposals.adoc)
+  first as a draft in this repository.
+* (everyone) Provide feedback on application text, reach consensus.
+* (russellb) Submit Application.
+* (russellb + others) Prepare and deliver a presentation to the CNCF TOC.
+  Invite metal3 community members to attend.
+* (russellb) If accepted, transfer project assets as appropriate to CNCF.
+
+## Alternatives
+
+### No Foundation
+
+We could stick with the status quo, where one person or organization owns
+certain assets (like domains), and evolve our governance as needed to support
+the collaboration among many different teams at different organizations.  While
+we seem to be working well together so far, we would also miss out on some of
+the benefits of being more closely aligned with the CNCF ecosystem.
```

Comment:
Could you list a couple benefits.

Reply:
You mention the site URL https://www.cncf.io/services-for-projects above.  It would be helpful if you mentioned a couple of the important benefits here.

Reply:
I'm not sure what to add, since this comment is under the "No Foundation" section?

Reply:
is not an appealing alternatve.